tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -216,7 +227,7 @@ public ImmutableTracerSettings(TracerSettings settings)
         /// Default is <c>false</c>.
         /// </summary>
         /// <seealso cref="ConfigurationKeys.LogsInjectionEnabled"/>
-        public bool LogsInjectionEnabled { get; }
+        public bool LogsInjectionEnabled => DynamicSettings.LogsInjectionEnabled ?? _logsInjectionEnabled;
```

Comment:
Oh no, this is going to be massively conflicting with [my pr](https://github.com/DataDog/dd-trace-dotnet/pull/4259/files) 😬

Reply:
[narrator] It did.